server/accounts.go

```diff
@@ -925,17 +951,67 @@ func buildInternalNkeyUser(uc *jwt.UserClaims, acc *Account) *NkeyUser {
 
 // AccountResolver interface. This is to fetch Account JWTs by public nkeys
 type AccountResolver interface {
-	Fetch(pub string) (string, error)
+	Fetch(name string) (string, error)
```

Comment:
Since exported, I would put a comment for each function.

Reply:
+1